src/Catalog/Ownership/StorageRegistration.cs

```diff
@@ -5,6 +5,7 @@
 using System;
```

Comment:
I think we should just drop the whole StorageRegistration class at this point. We have the interface and the table implementation; its tedious keeping this up to date.